docs/source/download.md

```diff
@@ -1,10 +1,10 @@
 # Download
 
-## v1.7.0 release
-
-Coming soon - end of December 2019
-* [RC1](https://github.com/openucx/ucx/releases/tag-v1.7.0-rc1)
+## v{RELEASE} release
 
+* Download [TGZ](https://github.com/openucx/ucx/releases/download/v{RELEASE}/ucx-{RELEASE}.tar.gz) [SRPM](https://github.com/openucx/ucx/releases/download/v{RELEASE}/ucx-{RELEASE}-1.fc30.src.rpm)
```

Comment:
Let's just use latest ?
https://help.github.com/en/github/administering-a-repository/linking-to-releases

I don't think that for user guide we have to reference exact version. I would rather generalize.

Reply:
then it will take the user to another site
I think it's more convenient to have direct links in this page
especially that we have to change the RELEASE tag in docs/conf.py (in one place) anyway

Reply:
But the "another" site is super simple to navigate with different download options.  Otherwise the document actually has to be generated from v1.7.x branch. IMHO this is unnecessary complexity and we will not remember to update this after each release.

Reply:
@shamisp actually there is something we are not doing right today: we are generating RTD from "master", but actually we should generate it for each branch. We should probably make html/pdf build as part of RTD build. RTD supports multiple "sites" per version/branch.
therefore, the master branch has hardcoded VERSION of the latest release (v1.7)

Reply:
i think we can merge this PR since we have to update the download links in RTD anyway, and fix the above in later PR.

Reply:
@yosefe regardless of methodology we are introducing one more place that we have to remember to bump up versioning. Since this is python script, can it fetch the version from GitHub (latest tag without RC or something like this:
```bash
curl --silent "https://api.github.com/repos/openucx/ucx/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/'
```

Reply:
```python
import subprocess
subprocess.call('curl --silent "https://api.github.com/repos/openucx/ucx/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/'', shell=True)
```

BTW, I'm sure there is better way to parse the text natively in python than grep/sed

Reply:
here we go...
https://docs.python.org/3/library/json.html

Reply:
@shamisp fixes, see https://github.com/openucx/ucx/pull/4715/files#diff-1ec61223727a10e5e4c74c68336809fcR46

Reply:
I guess there is python library literally for everything :)